DfciPkg/UnitTests/DfciTests/RefreshServer/Src/main.py

```diff
@@ -49,7 +49,7 @@ def hello_world():
         else:
             raise Exception(f'Unsupported format URL {request.url}')
         msg += '\nRequest from '
-        msg += request.remote_addr
+        msg += escape(request.remote_addr)  # sanitize user input

```

Comment:
Just taking a quick look at the file, I didn't see where `escape()` is defined in this context. Is this referring to [`re.escape()`](https://docs.python.org/3/library/re.html#re.escape)?

Reply:
no.  I think it is https://docs.python.org/3/library/html.html#html.escape but it has been so long i don't remember if i tried this or just followed the code ql feedback.  I agree it might need to be imported.